recipes/laravel-horizon/README.md

```diff
@@ -0,0 +1,32 @@
+# Laravel: Horizon
+
+This is an step-by-step guide to install [Laravel Horizon](https://laravel.com/docs/master/horizon) into your project.
+This expects the project to already have an working laravel project.
+
+## Steps
+
+1. Install redis using
+   the [service example](https://github.com/drud/ddev-contrib/blob/master/docker-compose-services/redis)
+2. Configure the redis configuration in `.env`
+3. Run `ddev composer require laravel/horizon`
+4. Run `ddev artisan horizon:install`
+5. Run horizon process in DDEV Installation
+
+    * Option A: Run using nohup
+
+      Use the `dot.ddev/config.laravel-horizon.yaml`
+
+    * Option B: Run using supervisor
+
+      Use both `dot.ddev/web-build/Dockerfile` and `dot.ddev/web-build/horizon.conf`
```

Comment:
This is a great example. Please add more detail on what you're doing here.  (Note that this can also be done using docker-compose.*.yaml to mount the horizon.conf)
```suggestion
      Copy the `dot.ddev/web-build/Dockerfile` and `dot.ddev/web-build/horizon.conf` into your project's .ddev/web-build directory. This causes the build process to add the starting of the horizon process to the startup of the container (managed by supervisord, which also starts nginx and php-fpm).
```

Reply:
Interesting suggestion, i have it locally like this;
`docker-compose.horizon.yaml`
```yaml
version: '3.6' 
services:
  web:
    volumes:
      - ./horizon/supervisord.horizon.conf:/etc/supervisor/conf.d/horizon.conf
```

Did you mean this by using docker-compose.*.yaml?

Reply:
Yes, that's what I meant. Using that technique is an alternative to doing the Dockerfile.  Both are fine.